hpx/components/containers/partitioned_vector/partitioned_vector_segmented_iterator.hpp

```diff
@@ -521,6 +521,11 @@ namespace hpx
           : base_type(it), data_(data)
         {}
 
+        operator BaseIter const &() const
```

Comment:
I meant, shouldn't this be:

    explicit operator BaseIter const& () { ... }

?